src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: lightgray;
```

Comment:
Consider using a more descriptive color value instead of 'lightgray'. Example: #f0f0f0 or rgba(240, 240, 240, 1)